src/components/Type/Type.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./Type.css";
+
+const Type = ({ pokemonItem, isLarge }) => {
+  let typesArray = pokemonItem?.types;
+  return (
+    <div className={isLarge ? "type-container-large" : "type-container-small"}>
```

Comment:
yeah we don't need to use two classes here cos they are the same anyway

Reply:
same cant